tests/libs/estdlib/test_gen_server.erl

```diff
@@ -316,6 +317,29 @@ test_timeout_info_repeats(Pid, Sleep) ->
             test_timeout_info_repeats(Pid, 2 * Sleep)
     end.
 
+test_timeout_tuple_return() ->
+    case get_otp_version() of
+        Version when Version =:= atomvm orelse (is_integer(Version) andalso Version >= 28) ->
```

Comment:
Version >= 28 would work since you converted OTP version to integer. Atoms are larger than integers (think infinity).

Reply:
So you can simplify with a comment.

% AtomVM and OTP >= 28 handle timeout tuple
Version when Version >= 28 ->